lib/builders/observability-builder.ts

```diff
@@ -0,0 +1,77 @@
+import * as blueprints from '../../lib';
+import * as utils from "../utils";
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ObservabilityBuilder extends blueprints.BlueprintBuilder {
+
+    public addNativeObservabilityBuilderAddOns(): ObservabilityBuilder {
+        return this.addOns(
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
+            new blueprints.addons.CertManagerAddOn(),
+            new blueprints.addons.CoreDnsAddOn(),
+            new blueprints.addons.KubeProxyAddOn(),
+            new blueprints.addons.KubeStateMetricsAddOn(),
+            new blueprints.addons.MetricsServerAddOn(),
+            new blueprints.addons.PrometheusNodeExporterAddOn(),
+            new blueprints.addons.VpcCniAddOn());
+    }
+
+    public addMixedObservabilityBuilderAddOns(): ObservabilityBuilder {
+        return this.addOns(
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
+            new blueprints.addons.AdotCollectorAddOn(),
+            new blueprints.addons.CertManagerAddOn(),
+            new blueprints.addons.CoreDnsAddOn(),
+            new blueprints.addons.KubeProxyAddOn(),
+            new blueprints.addons.KubeStateMetricsAddOn(),
+            new blueprints.addons.MetricsServerAddOn(),
+            new blueprints.addons.PrometheusNodeExporterAddOn(),
+            new blueprints.addons.VpcCniAddOn());
+    }
+
+    public addOpenSourceObservabilityBuilderAddOns(): ObservabilityBuilder {
```

Comment:
Why not add AMP here, optionally accepting the config for it?

Reply:
remove